### PR TITLE
区分汉语英语

### PR DESCRIPTION
**Pull request by Kas1o**

### CODE REVIEW

File: App.cs [hunk 1]
```diff
@@ -170,6 +176,7 @@ private static async Task MethodPlanA()
 
             if (fLeagueClientZoom == -1)
             {
+                if (bIsChinese) Console.WriteLine("无法设置 LeagueClientUx 的原始缩放");
```
Comment (BakaFT) [inline App.cs:179]: `get` => `设置`

🤔are you sure?

Comment (Kas1o) [reply App.cs:179]: 🤯

Comment (Kas1o) [reply App.cs:179]: Even my commit info  *Localization* =>*Localzetion* is wrong spell, what happens to my brain last hour?

Comment (Kas1o) [reply App.cs:179]: and my commit info Localization =>Localzetion is wrong spell, what happens to my brain last hour?

Comment (Coooookies) [reply App.cs:179]: > and my commit info Localization =>Localzetion is wrong spell, what happens to my brain last hour?

This project is about to be reset, you can make the Pull request later. :D
这个项目将会重构，你可以稍后再提交PR。:D

**Pull request closed without merging**